components/count-badge/demo/count-badge-icon.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<meta http-equiv="X-UA-Compatible" content="ie=edge">
+		<meta charset="UTF-8">
+		<link rel="stylesheet" href="../../demo/styles.css" type="text/css">
+		<script src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
```

Comment:
Don't need the `webcomponents-loader.js` or `<meta http-equiv="X-UA-Compatible" content="ie=edge">` lines anymore now that IE and legacy-Edge are gone.